tests/templates/kuttl/smoke/60-install-hive.yaml.j2

```diff
@@ -15,9 +15,8 @@ spec:
   clusterConfig:
     database:
       connString: jdbc:postgresql://postgresql:5432/hive
-      user: hive
-      password: hive
       dbType: postgres
+      credentialsSecret: hive-credentials
```

Comment:
Lets move `credentialsSecret` up as in other examples/tests?

Reply:
Done: 67ed2e0